packages/core/README.md

```diff
@@ -18,9 +18,9 @@ This repository contains the static HTML, Javascript, CSS, and images content of
 
 -   [Modules docs](docs/Modules/README.md) - Contains implementation guides (i.e., scaffolding help, etc.)
 
-## Use a custom domain
+## Manual deployment using custom domain (CNAME)
 
-To use your custom domain, please put it in a file named `CNAME` inside the `scripts` folder of your local clone of deriv-app.
+For manual deployment, it's necessary to use the file named `CNAME`, please put a custom domain to this file. It's only for the gh-pages branch of your local clone of deriv-app.
```

Comment:
where do we have to put the file?
can you include that in your description?
also for the last part
i think its better to have something like:
`It's only for the deployment on github pages for your forked repo of deriv-app.`